### ISSUE TITLE
False negatives in no-explicit-any

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno_lint#156]
```
(ban-unused-ignore) Ignore for code "no-explicit-any" was not used.
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/node/module.ts:104:4
```
```ts
  static _extensions: {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    [key: string]: (module: Module, filename: string) => any;
  } = Object.create(null);
```

```
(ban-unused-ignore) Ignore for code "no-explicit-any" was not used.
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/node/module.ts:1048:2
```
```ts
type RequireWrapper = (
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  exports: any,
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  require: any,
  module: Module,
  __filename: string,
  __dirname: string
) => void;

```

**Comment by bartlomieju** [comment]
The actual problem lies in the fact that there are multiple ignore directives created each of the comments - should be fixed by https://github.com/denoland/deno_lint/pull/187
